dojson/utils.py

```diff
@@ -96,8 +96,12 @@ def wrapper(self, key, values, **kwargs):
 
 def force_list(data):
     """Wrap data in list."""
-    if data is not None and not isinstance(data, (list, tuple, set)):
-        return (data,)
+    if data is None:
+        return []
```

Comment:
The thing is that it would improve so much client code if we can rely on having on this function to always return a list. In this way one can always safely use it in for loops without always having the inelegant pattern:

```python
for foo in force_list(...) or []:
    ...
```